src/data_management/labels/LabeltypesTable.tsx

```diff
@@ -22,7 +22,7 @@ const columnDefinitions = [
         className={tableStyles.CellEllipsis}
         title={row.name}
       >
-        <NavLink to={`${navigationUrl}/${row.uuid}/`}>{!row.name? "(empty name)" : row.name }</NavLink>
+        <NavLink to={`${navigationUrl}/${row.uuid}`}>{!row.name? "(empty name)" : row.name }</NavLink>
```

Comment:
Removed the "/" at the end of the URL to remove the symbol ">" at the end of the navigation bar